tests/function_tests/test_common.py

```diff
@@ -0,0 +1,71 @@
+'''
+  Copyright 2017, Dell, Inc.
+
+  Author(s):
+
+  UCS test common methods and functions
+
+'''
+
+import os
+import sys
+import json
+import requests
+
+requests.packages.urllib3.disable_warnings(
+    requests.packages.urllib3.exceptions.InsecureRequestWarning
+)
+requests.packages.urllib3.disable_warnings(
+    requests.packages.urllib3.exceptions.InsecurePlatformWarning
+)
+requests.packages.urllib3.disable_warnings(
+    requests.packages.urllib3.exceptions.SNIMissingWarning
+)
+sys.path.append("/".join((os.path.split(os.path.realpath(__file__))[0]).split('/')[:-2]))
+
+from util import util  # noqa: E402
+
+UCSM_CONFIG = json.loads(os.getenv(
+    'UCSCONFIG',
+    '''
+    {
+        "ucs-host": "10.240.16.144",
```

Comment:
It's not secure to expose the IP

Reply:
Good finding, I will remove default UCSM configure.